microservice_demo/README.md

```diff
@@ -0,0 +1,36 @@
+# Kernel Gateway Microservice Demonstration
+
+The [Kernel Gateway](https://github.com/jupyter-incubator/kernel_gateway) is a
+[JupyterApp](https://github.com/jupyter/jupyter_core/blob/master/jupyter_core/application.py) that
+implements different APIs and protocols for accessing Jupyter kernels.  We can use the Jupyter kernel gateway to
+[process requests](https://github.com/jupyter-incubator/kernel_gateway#processing-requests) and transform an
+annotated notebook into a HTTP API using the Jupyter kernel gateway.
+ 
+This example shows how one can use the Jupyter kernel gateway to deploy an notebook as a
+microservice at the heart of a Twitter bot, using a combination of [Alchemy API](http://www.alchemyapi.com/products/alchemylanguage/entity-extraction) with IFTTT's [Twitter](https://ifttt.com/twitter) and [Maker](https://ifttt.com/maker) channels, to respond to mentions with a search URL for relevant Meetups. You will need developer keys for [Alchemy](http://www.alchemyapi.com/api/register.html) and [Maker](https://ifttt.com/maker) to proceed, as well as an arbitrary Event Name for use with Maker.
+
```

Comment:
Include the name of the running bot and the API endpoint so people can try the live thing without deploying it themselves.